File: emarsys/src/androidTest/java/com/emarsys/testUtil/ExperimentalTestUtils.java
package com.emarsys.testUtil;

import com.emarsys.core.experimental.ExperimentalFeatures;

public class ExperimentalTestUtils {

    private ExperimentalTestUtils() {
    }

    public static void resetExperimentalFeatures() {
        ReflectionTestUtils.invokeStaticMethod(ExperimentalFeatures.class, "reset");
    }
}

File: emarsys/src/androidTest/java/com/emarsys/testUtil/ExperimentalTestUtils.kt
package com.emarsys.testUtil

import com.emarsys.core.experimental.ExperimentalFeatures

object ExperimentalTestUtils {

    @JvmStatic
    fun resetExperimentalFeatures() {
        ReflectionTestUtils.invokeStaticMethod(ExperimentalFeatures::class.java, "reset")
    }
}
